routeros/resource_system_logging.go

```diff
@@ -0,0 +1,73 @@
+package routeros
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+/* {
+    ".id": "*4",
+    "action": "echo",
+    "default": "true",
+    "disabled": "false",
+    "invalid": "false",
+    "prefix": "",
+    "topics": "critical"
+} */
+
+var validTopics = []string{
```

Comment:
Were you planning to do validation?

Reply:
My bad: I implemented validation but terraform does not support validation for lists :( and I only found out after it was implemented (error message on the sdk), I deleted the function but not the slice. 

I've just deleted the slice.

Reply:
Actually I found a way to add the validation.